runtime/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I suggest we use the log4j.properties file. Its much more readable. 
@regunathb - makes sense?